src/GrpcProtos/capture.proto

```diff
@@ -50,6 +50,7 @@ message CaptureOptions {
     kDwarf = 2;
   }
   UnwindingMethod unwinding_method = 4;
+  uint32 stack_dump_size = 16;
```

Comment:
I would add a comment on the limit of this (since protos don't have `uint16`).

Reply:
Done.